tests/test_quick_run_config_generator.py

```diff
@@ -244,20 +244,29 @@ def test_get_next_run_config_multi_model(self):
         self.assertEqual(pc2['batch-size'], 1)
         self.assertEqual(pc2['model-version'], 3)
 
-    def test_radius_magnitude(self):
-        """ 
-        Test that get_radius and get_magnitude work correctly 
+    def test_radius(self):
+        """
+        Test that get_radius works correctly.
 
         Expected radius is 6 (search config has 5, and offset is 1)
-        Expected magnitude is 8 (search config has 7, and offset is 1)
         """
 
-        urcg = self._urcg
-        urcg._radius_offset = 1
-        urcg._magnitude_offset = 1
+        qrcg = self._qrcg
+        qrcg._radius_offset = 1
+        self.assertEqual(qrcg._get_radius(), 6)
+
+    def test_magnitude(self):
+        """
+        Test that _get_magnitude works correctly.
+        """
+        qrcg = self._qrcg
+        self.assertEqual(qrcg._get_magnitude(), 7)  # initial value
+
+        qrcg._magnitude_scaler *= 0.5
+        self.assertAlmostEqual(qrcg._get_magnitude(), 3.5)
 
-        self.assertEqual(urcg._get_radius(), 6)
-        self.assertEqual(urcg._get_magnitude(), 8)
+        qrcg._magnitude_scaler *= 0.2
```

Comment:
just set it to 0.1 instead of multiplying. Otherwise it is confusing

Reply:
But that isn't what this is testing. You're coupling the two, and making the test confusing as a result.

What you are testing here is how get_magnitude is related to the magnitude_scalar. We don't care how or why magnitude_scalar is updated.

Reply:
Gotcha. Changed to setting the magnitude_scalar to a value instead of multiplying.